cyences_app_for_splunk/bin/user_inventory_v2_util.py

```diff
@@ -376,6 +388,9 @@ def _create_new_user(self):
 
     def add_user_entry(self, new_entry: UserEntry):
         # Assign a unique user_uuid to the user
+        if is_user_ends_with(new_entry.user, user_ends_with=["$"], user_postfixes=self.user_postfixes):
+            return "n/a"
```

Comment:
Why are we returning "n/a" instead of None or false or raise Exception?

Reply:
return value will be displayed as user_id field (random uuid) on search output.
In case of user ends with $, we ignoring the entry hence it will display user_id as "n/a". In case of None and False, it will display as blank and 0 respectively.